pkg/kv/store_message.go

```diff
@@ -14,54 +13,46 @@ type StoreMessage struct {
 }
 
 func (s *StoreMessage) GetMsg(ctx context.Context, path string, msg protoreflect.ProtoMessage) error {
-	val, err := s.Store.Get(ctx, []byte(path))
+	v, err := s.Store.Get(ctx, []byte(path))
 	if err != nil {
-		return fmt.Errorf("failed on Get (path: %s): %w", path, err)
+		return err
```

Comment:
Why did you remove the formatting? It was the only place where we log the key that failed